LabBuilder/LabBuilder.psm1

```diff
@@ -4762,6 +4880,200 @@ Function Uninstall-Lab {
+            Position=1)]
+	   [System.Collections.Hashtable[]] $VMTemplateDisks
+    )
+
+
+	$ScriptDir = $Config.labbuilderconfig.settings.FullConfigPath
+	$TemplateDiskPath = $Config.labbuilderconfig.settings.TemplateDiskPath
+	$ServerISO = $Config.labbuilderconfig.settings.TemplateISO
+
+	[String]$WorkFolder = Join-Path -Path $ScriptDir -ChildPath 'WinImage'
+	[String]$DismFolder = Join-Path -Path $ScriptDir -ChildPath 'DISM'
+	[String]$MountFolder = Join-Path -Path $ScriptDir -ChildPath 'Mount'
+
+	Write-Verbose "Script Directory = $ScriptDir"
+	Write-Verbose "Template disk path = $TemplateDiskpath"
+	Write-Verbose "ISO to use = $ServerISO"
+
+
+	##########################################################################################
+	#   Create Parent VHDX files for VMs
+	##########################################################################################
+	# Mount the Windows Server 2016 ISO and get the drive letter
+
+
+	Write-Verbose -Message 'Mounting Server ISO'
+	$null = Mount-DiskImage -ImagePath $ServerISO
+	[String]$DriveLetter = (Get-Diskimage -ImagePath $ServerISO | Get-Volume).DriveLetter
+	$ISODrive = "$([string]$DriveLetter):"
+
+	Write-Verbose "CD Drive is $ISODrive"
+	Get-PSDrive -PSProvider FileSystem # Work around an issue with script not seeing the drive 
+
+	Write-Verbose 'Copying DISM from Server ISO to Working Folders'
+	If (-not (Test-Path -Path $DismFolder -PathType Container)) 
+		{
+		$null = New-Item -Path $DismFolder -ItemType Directory
+		} #EndIf
+	$null = Copy-Item -Path "$ISODrive\Sources\api*downlevel*.dll" -Destination $DismFolder -Force
+	$null = Copy-Item -Path "$ISODrive\Sources\*dism*" -Destination $DismFolder -Force
+	$null = Copy-Item -Path "$ISODrive\Sources\*provider*" -Destination $DismFolder -Force
+	
+	
+
+	 # As of 2015-06-16 Convert-WindowsImage contains a function instead of being a standalone script.
+	 # . source the Convert-WindowsImage.ps1 so it can be called
+	. "$ScriptDir\Convert-WindowsImage.ps1"
+	$TemplatePrefix  = $Config.labbuilderconfig.TemplateDisks.Prefix
+
+	Write-Verbose "Prefix to use $TemplatePrefix"
+
+	# Create working folder
+	Write-Verbose -Message 'Creating Working Folders'
+	If (-not (Test-Path -Path $WorkFolder -PathType Container)) 
+		{
+		$null = New-Item -Path $WorkFolder -ItemType Directory
+		} #EndIf
+
+	foreach ($VMTemplateDisk in $VMTemplateDisks)
+		{
+			$VMTemplateDiskName = "$TemplatePrefix" + [string]$VMTemplateDisk.Name
+			Write-Verbose "Disk to be created = $VMTemplateDiskName"
+
+			$PathtoDisk = $TemplateDiskpath + "\" + $VMTemplateDiskName
+
+			if (!(Test-Path $PathtoDisk ))
+			{ 
+				[String]$VHDFormat = $VMTemplateDisk.VHDFormat
+				If ($VHDFormat -eq $null) {$VHDFormat = 'VHDX'}
+
+				Write-Verbose "VHD Format is $VHDFormat"
+
+				$WimpathTestVariable = [string]$VMTemplateDisk.isNano
+
+				Write-Verbose "Current Nano Setting is $WimpathTestVariable"
+
+				If ($VMTemplateDisk.isNano -eq $null){$WimPath = 'Sources\Install.WIM'}
```

Comment:
Can we change the expected values of the isNano parameter in the XML to either 'Y' or 'N'? this keeps it consistent with the other XML Y/N fields in XML. It may be confusing to use Y/N someplaces and True/False in others.